Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -147,7 +147,11 @@ public override async Task SetParametersAsync( ParameterView parameters )
                 if ( !IsMultiple && CurrentSearch != SelectedText && !string.IsNullOrEmpty( SelectedText ) )
                 {
                     currentSearch = SelectedText;
-                    await CurrentSearchChanged.InvokeAsync( currentSearch );
```

Comment:
Isn't there a SetSearch helper? With both the new parameter and the obsolete?

Reply:
Apparently, there is. You can look at it for 1.1.2?

Reply:
Kk